docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+
+  - `urn:x-nmos:cap:meta:preference` 
+    - set to 100 if the video mode is a preferred mode.  
+    - set to -100 if the video mode is a non-native mode.  
+    - set to 0 all others  
+
+- MUST not add a `contraints_set` for any video mode the device cannot support due to implementation limitations even though the display supports the mode.
+
+- MUST add a grouping tag to the created video receiver with tag `name` and `role` set to indicate a group comprised of the video receiver and audio receiver associated with the display.
+
+- SHOULD provide a `contraint_sets` entry for each of the video modes supported by the display as provided by the EDID as an individual `contraint_set`on the receiver with parameters set as described above if the mode is supported by the NMOS device.
+
+##### Audio Mappings
+
+In order that a NMOS device that provides Audio Receivers can work with EDID type audio devices the device:
+
+- MUST create one IS-04 Audio Receiver for each of the displays associated with the IS-04 Device that support audio.
+
+- MUST set the `format` element to the value `urn:x-nmos:format:audio`  
+
+- MUST associate with each IS-04 Audio Receiver a `contraints_set` resource in the IS-04 Receiver's `caps` resource  as described in [BCP-004-1]
+
+- MUST map at least one of the audio modes present by the display device's EDID into a `contraint_set` with the following elements set:
+
+  - `urn:x-nmos:cap:meta:enabled` set to `true`.  
+
+  - `urn:x-nmos:cap:format:channel_count` set to number of audio channels as taken from EDID's CEA-861-EXTENSION-Block's Audio Data element.
+
+  - `urn:x-nmos:cap:format:channel_count` set to number of audio channels as taken from EDID's CEA-861-EXTENSION-Block's Audio Data element.
+
+  - `urn:x-nmos:cap:format:sample_rate` set to array of supported sample rates as taken from EDID's CEA-861-EXTENSION-Block's Audio Data element.
+
+  - `urn:x-nmos:cap:format:sample_depth` set to array of supported sampling bit rates as taken from EDID's CEA-861-EXTENSION-Block's Audio Data element.
+
+- MUST add a grouping tag to the created audio receiver with tag `name` and `role` set to indicate a group comprised of the video receiver and audio receiver associated with the display.
+
+- SHOULD provide a `contraint_sets` entry for each of the audio modes supported by the display as provided by the EDID as an individual `contraint_set`with parameters set as above.
+
+### EDID to NMOS IS-EDID-CM Resource
+
+IS-EDID-CM provides an API for the IS-04 EDID Device Control.  The following items apply to an NMOS device that has EDID type displays:
+
+- MUST maintain a binary representation of the EDID for each display and support GET for the endpoint `edid-cm\receiver-id\edid` where receiver-id is the IS-04 UUID for the receiver.
+
+- For each Receiver 
+- MUST populate the `edid-cm\destination` resource with the following parameters:
```

Comment:
What is behavior for a decoder gateway with multiple different displays connected? We need to end up #27 to clarify this.